background.ts

```diff
@@ -226,6 +227,7 @@ Electron.ipcMain.handle('service-forward', async(event, service, state) => {
 const adjustNameWithDir: Record<string, string> = {
   helm:    path.join('bin', 'helm'),
   kubectl: path.join('bin', 'kubectl'),
+  kim:     'kim',
```

Comment:
If there's no change, we don't really need it in the mapping?

Reply:
done